wp-parsely.php

```diff
@@ -188,6 +186,17 @@ function maybe_load_plugin() {
 		return;
 	}
 
+	$is_disabled_via_constant = defined( 'VIP_PARSELY_SKIP_LOAD' ) && true === constant( 'VIP_PARSELY_SKIP_LOAD' );
+	$is_fedramp               = defined( 'VIP_IS_FEDRAMP' ) && true === constant( 'VIP_IS_FEDRAMP' );
```

Comment:
For the `VIP_IS_FEDRAMP` constant, we need to a take a similar approach to Jetpack: https://github.com/Automattic/vip-go-mu-plugins/blob/9fead4d530e81177b7efaec5acf32588ee37fd62/000-vip-init.php#L131-L134

I would put something like this next to the Jetpack one for now:

```
if ( Context::is_fedramp() ) {
    if ( ! defined( 'VIP_PARSELY_SKIP_LOAD' ) ) {
        define( 'VIP_PARSELY_SKIP_LOAD', true );
    }
}
```

By default, this will skip loading Parse.ly. But, FR customers can override the behaviour by defining the constant as false in their vip-config.php file (`define( 'VIP_PARSELY_SKIP_LOAD', false );`, which allows them to use the plugin.

Reply:
I was initially doing it this way but was not sure how to properly test this. Based on the feedback I am incorporating this change but please let me know how can I write test for it properly .... any existing example that can tests the logic we are going to put in `000-vip-init.php` file?

Reply:
I think the a good approach would be to move the code to a function and then test the function. 

@sjinks might have some better suggestions as well.

Reply:
`Context::is_fedramp()` is already a function ....... but how can I mock its output inside my test.

It is getting called by `000-vip-init.php` file so how can I mock it 🤔

Reply:
For now I am merging the PR, will add the test once I get the response or figure out the way myself.

Reply:
Take a look at `Constant_Mocker` (tests/files/acl/test-acl.php) or the `createMock` method (tests/admin-notice/test-class-admin-notice.php).

Reply:
I did looked into those but those didn't worked and most probably the reason is that in `000-vip-init.php` file is getting called very early (also in tests) which means this setting up `VIP_PARSELY_SKIP_LOAD` constant decision is taking place very early so these mocks or functions cannot help.